src/components/20-molecules/footer-small/index.js

```diff
@@ -0,0 +1,67 @@
+import { LitElement, html, css, unsafeCSS } from 'lit-element';
+import footerSmallCSS from './index.scss';
+
+class AXAFooterSmall extends LitElement {
+  static tagName = 'axa-footer-small';
+  static styles = css`
+    ${unsafeCSS(footerSmallCSS)}
+  `;
+
+  static get properties() {
+    return {
+      languageLinks: { type: Array },
+      disclaimerLinks: { type: Array },
+      copyrightText: { type: String },
+    };
+  }
+
+  constructor() {
+    super();
+    this.languageLinks = [];
+    this.disclaimerLinks = [];
+    this.copyrightText = '';
+  }
+
+  formatText(text, arrIndex, arrLength) {
+    return `${text}${arrIndex !== arrLength - 1 ? ' | ' : ''}`;
```

Comment:
@raphaellueckl please use array method join instead. Reason: better readability

https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Global_Objects/Array/join

https://jsbin.com/kemifiwuhe/edit?js,console

Reply:
And better would be use a pseudo element for pipeing as the pipe is a visual or design element  and not part of text.

```
.m-footer-small__link + .m-footer-small__link::before {
    content: " | ";
}
```

Reply:
Plus relying on the index of map is dangerous as you make 2 function hardly coupled.

Reply:
@LucaMele I learned about this method with css this weekend :D Some guy made a compilation of css tricks, I didn't know that before, but I will go for that one, thanks!